<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sigmaX_text">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma X</string>
    </property>
   </widget>
   <widget class="QLabel" name="sigmaY_text">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="canny_text">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny threshold 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="canny_text_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny threshold 2</string>
    </property>
   </widget>
   <widget class="QDial" name="sigmax_dial">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="sigmay_dial">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="canny_inf_dial">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="canny_sup_dial">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="sigmax">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="sigmay">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="canny_inf">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="canny_sup">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>440</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cap">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>351</width>
      <height>251</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="filter_video">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>351</width>
      <height>251</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
